Eyansky/StackOverflow-lite
# #159729375 Api get all questions

#### What does this PR do?
- As a User, I should be able to get all the questions asked.

#### Description of Task to be completed?
- Creating endpoint to get all questions.

#### How should this be manually tested?
- Clone this repo
```
- git clone https://github.com/Eyansky/StackOverflow-lite
```
- Navigate to this branch 
```
- git checkout ft-get-all-questions-159729375
```
- Open the Api folder
```
- cd Api
```
- Create a virtual environment and install dependencies from the requirements.txt
```
- virtualenv env
- pip install -r requirements.txt
```
- run the app
```
- python run.py
```
- Use postman to check if this 'GET' endpoint is working as expected.
```
http://127.0.0.1:5000/api/v1/users/questions
```
#### Any background context you want to provide?
This is only the GET endpoint to collect all questions.
All data is in form of JSON.
The questions data is hard coded into the api.

#### What are the relevant pivotal tracker stories?
- #159729375
#### Screenshots (if appropriate)
<img width="1680" alt="screen shot 2018-08-16 at 17 51 51" src="https://user-images.githubusercontent.com/26874561/44216260-32333200-a17d-11e8-9be9-2ca84ca281a6.png">
